patterns/contact-centered-social-link.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Title: Centered link and social links
+ * Slug: twentytwentyfive/contact-centered-social-link
+ * Keywords: contact, social-link
+ * Categories: contact
+ *
+ * @package Twenty Twenty-Five
+ */
+
+?>
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--80);padding-bottom:var(--wp--preset--spacing--80)">
+	<!-- wp:paragraph {"align":"center","fontSize":"x-large"} -->
+	<p class="has-text-align-center has-text-color has-link-color has-x-large-font-size"><?php echo wp_kses_post( _x( 'Got questions? <br><a href="#" rel="nofollow">Feel free to reach out.</a>', 'Heading of the Contact social link pattern', 'twentytwentyfive' ) ); ?></p>
```

Comment:
Since the text and link color was removed, the class names must also be removed:
has-text-color has-link-color

Reply:
Updated the code.